src/main/java/com/wire/bots/hold/service/DeviceManagementService.java

```diff
@@ -164,6 +165,18 @@ public void confirmDevice(QualifiedId userId, UUID teamId, String clientId, Stri
      * @throws CryptoException
      */
     public void removeDevice(QualifiedId userId, UUID teamId) throws IOException, CryptoException {
+        // MLS
+        LHAccess userAccess = accessDAO.get(userId.id, userId.domain);
+        if (userAccess != null) {
+            API api = new API(client, null, userAccess.token);
+            if (api.isMlsEnabled()) {
+                try (CryptoMlsClient cryptoMlsClient = new CryptoMlsClient(userAccess.clientId, coreCryptoPassword)) {
+                    cryptoMlsClient.wipe();
+                }
```

Comment:
This should basically call close() twice, once for the try-with-resources and the second time inside the close().
Normally it should be okay but there is a misterious coment inside the CoreCrypto destroy method:
```
override fun destroy() {
        // Only allow a single call to this method.
        // TODO: maybe we should log a warning if called more than once?
        if (this.wasDestroyed.compareAndSet(false, true)) {
            // This decrement always matches the initial count of 1 given at creation time.
            if (this.callCounter.decrementAndGet() == 0L) {
                cleanable.clean()
            }
        }
    }

    @Synchronized
    override fun close() {
        this.destroy()
    }
```
However even if the comment says that, there is a boolean that says: if the instance is already destroyed, do nothing, which is good.
Anyway, if your test runs, it means that calling close() twice is not an issue